rest-api-spec/src/main/resources/rest-api-spec/test/search/180_locale_dependent_mapping.yml

```diff
@@ -1,8 +1,5 @@
 ---
 "Test Index and Search locale dependent mappings / dates":
-  - skip:
-      version: " - 6.1.99"
```

Comment:
Is this redundant now?

Reply:
so in master I think it does not make sense. The ` - 6.1.99` is translated into `7.0.0 - 6.1.99` And master runs bwc tests from 7.0 - 8 range